index.html

```diff
@@ -8216,6 +8217,107 @@ <h5>Presentational Roles Conflict Resolution</h5>
 				</tbody>
 			</table>
 		</div>
+		<div class="role" id="suggestion">
+			<rdef>suggestion</rdef>
+			<div class="role-description">
+					<p>A single proposed change to content that can be approved or rejected.</p>
+					<p>For example, in an editing system that supports multiple authors, one author may suggest a change, and another author would be responsible
+					for accepting or rejecting the suggestion.
+					The suggested change MUST be present as child content, as an <rref>insertion</rref>, a <rref>deletion</rref>, or one of each. A suggestion MUST NOT contain
+				  any other children. Additional information such as related comments, authoring info and time stamps MAY be provided via aria-description or aria-details.</p>
```

Comment:
```suggestion
				  any other children. Additional information such as related comments, authoring info and time stamps MAY be provided via <pref>aria-description</pref> or <pref>aria-details</pref>.</p>
```

Reply:
Can't decide if we would need to have ` on the element with role <rref>suggestion</rref>` at the end of the sentence with aria-details, or if it's obvious from context and that would just be too many words.